### ISSUE TITLE
Where to post?

### ISSUE THREAD

**Issue by RalfRoemling** [soerennb/extplorer#19]
Hi Soeren,
found multiple contact options while researching. Wouldn't it be a good idea to reduce these to one?

I see:
- https://extplorer.net/projects/extplorer/boards
- https://extplorer.net/projects/extplorer/issues
- https://github.com/soerennb/extplorer/issues

All the best
Ralf

**Comment by estradis** [comment]
To be honest, the project doesn't look particularly well managed to me and the operators don't seem particularly communicative either.
- There are open tickets from 2012 (10 years old)
- No moderation of the posts (spammers can post as they like).
- No answers to open questions
- No reaction to submitted bugs
- No indication of what exactly is (or isn't) currently in progress

No clear direction is apparent to me. Rather I have the impression, Sören develops the project mainly for himself and puts it public only because he has no time (or desire) to look for reported errors himself. That should then be done by the community.

I like the project, but each update costs us an incredible amount of time to develop all the bugs out on a test server before we can use it productively. Therefore we are always 2-3 versions behind. Probably in a few years the project won't play a role for us anymore.

Meanwhile I consider to pull a standalone fork myself to hopefully do it better.

**Comment by RalfRoemling** [comment]
Yea I like Extplorer a lot too and that for a loooong time too. Thx to @soerennb for his work so far.

And thx for your opinion. I havn't been into the subject this deep. I'm only a user and have alerts running on "Extplorer" and "CVE"s for security.

Would really be great if you'd fork this so the community could get more support when Soeren is to occupied to do it himself.

**Comment by estradis** [comment]
Something seems to have changed! Spammer issues are now moderated out and are no longer available. Unfortunately, everything else has remained the same. It was a start, but probably already an end either.

In the meantime, the decision has been made at executive level that we will look for an alternative. This also means that there will be no fork from our side. 

The main reasons for the decision were that we cannot understand some individual code sections due to a lack of reasonable documentation, but also because we are no longer willing to invest any more time in research.

**Comment by RalfRoemling** [comment]
Ah, sad. Maybe you can post when you find an alternative? That'll be great. Thx a lot in advance! :-)

**Comment by atesin** [comment]
> Ah, sad. Maybe you can post when you find an alternative? That'll be great. Thx a lot in advance! :-)

here is a reasonable alternative = https://www.dulldusk.com/phpfm/

**Comment by RalfRoemling** [comment]
Wow, leaner cleaner meaner :-) Thxalot 👍 
I don't see any usecase for using Extplorer any further then. Do you?

**Comment by atesin** [comment]
i still like and use extplorer...  i also like phpfm, have some features extplorer dont (and vice versa), but has some little annoying bugs... check out issues page and patch yourself

**Comment by atesin** [comment]
i still like and use extplorer...  i also like phpfm, have some features extplorer dont (and vice versa), but has some little annoying bugs... check out github issues page and patch yourself

**Comment by atesin** [comment]
hi @soerennb , i am glad you are here

have you communicate with some forkers ?? ..  some of them had tried to patch/update this wonderful webapp

**Comment by soerennb** [comment]
No, I haven't done that yet, but it would be great if this project became the official fork...

**Comment by RalfRoemling** [comment]
Hi Soeren, 
welcome back. Great to see you're still with us :-)
I'm no developer just a user, so I guess I'm of little help here :-(

Have a great weekend everybody!

**Comment by estradis** [comment]
Hello @soerennb, 

glad to see you here either! Above all the irritations, you've developed a great product. It just needs a little more love from the developers. And - I'm sorry to say that - you need to be a bit more communicative as well. Please reply more often even if you are lacking of time.

We have also tried alternatives, but so far extplorer is the only one that meets our very strict requirements.
Our most important requirements are:

- Customers are only allowed to have download and upload folders, but they are not allowed to change them themselves and they are not allowed to create new folders.
- customers are only allowed to upload or create files in the upload folder, but not in other folders
- Customers are allowed to edit or delete their own files.
- Employees are allowed to create, edit and delete files in the download and upload folders, but they are not allowed to create subfolders.
- Administrators are allowed to do everything.
- The BLOB storage is outsourced to a back device. 
- Extplorer should show a banner when maintenance is scheduled or running.
- Extplorer should show a warning if logins are not possible (e.g. if the connection to the backend is lost)
- Extplorer shall send an email to specific users when an upod has taken place.
- and much more.

To realize this, considerable changes were necessary, incl. a prepend handler, because due to lack of documentation we couldn't find a central place how to make additional variables accessible to Extplorer before the initial loading of the pages.

This is also one of the reasons why we are constantly behind the current versions.

Although it has been decided that we as a company want to replace the product, there is no roadmap for this yet. It could also be possible that we will continue to run it further. I'll also try to get more involved privately, but time is tight there too right now.

@ the community:
A filebrowser with better experience for mobile devices
https://github.com/hurlenko/filebrowser-docker
